site/content/blogs/release-v122.en.md

```diff
@@ -35,6 +35,58 @@ Copilot v1.22 brings several new features and improvements:
 ## IAM Role Permissions Boundary
 
 ## FIFO SNS/SQS
+To have strict message ordering and message deduplication for your publish-subscribe architecture, you can now use SNS FIFO Topics and SQS FIFO Queues.
+
+### You can configure your manifest to have SNS FIFO Topics for your services.
+
+You can specify `fifo: true` in your service manifest under `publish.topics` as shown below and Copilot will create a FIFO SNS topic for you.
+
+```yaml
+publish:
+  topics:
+    - name: mytopic
+      fifo: true
+```
+
+Alternatively, you can also specify the advanced FIFO SNS Topic configuration as:
+```yaml
+publish:
+  topics:
+    - name: mytopic
+      fifo:
+        content_based_deduplication: true
+```
+
+### SQS FIFO queues for worker services
+You can specify `fifo: true` in your worker service manifest under `subscribe.topics.queue` or `subscribe.queue` as shown below and Copilot will create a FIFO SQS Queue and Subscriptions for you.
+
+```yaml
+subscribe:
+  topics:
+    - name: mytopic
+      service: myservice
+      queue: 
+        fifo: true # topics specific SQS FIFO Queue
+  queue:
+    fifo: true # default SQS FIFO Queue
```

Comment:
I think we *might* want to be clear that this is configuring the one and only one default queue to be FIFO, not a default queue for the fifo topics (meaning that there could be a default standard queue along side which is not the case). 

Besides we can probably lower-case "Queue"..?